src/homeTaskLwsson2/HomeTaskArr.java

```diff
@@ -0,0 +1,151 @@
+package homeTaskLwsson2;
+
+import java.util.Arrays;
+
+public class HomeTaskArr {
+    // сделал чисто int вариант, в техническом задание про дженеричность ничего нет
+
+    public int[] arr = new int[100000];
+    int size = 0;
+
+    boolean isSorted = false;
+
+    public static void main(String[] args) {
+
+        HomeTaskArr homeTaskArr = new HomeTaskArr();
+        rundomMath(homeTaskArr);
+
+        System.out.println("------------");
+        long start = System.currentTimeMillis();
+        homeTaskArr.sortBubble();
+        long finish = System.currentTimeMillis();
+        long timeConsumedMillis = finish - start;
+        System.out.println(timeConsumedMillis);
+
+        System.out.println("------------");
+        rundomMath(homeTaskArr);
+        long start1 = System.currentTimeMillis();
+        homeTaskArr.sortSelect();
+        long finish1 = System.currentTimeMillis();
+        long timeConsumedMillis1 = finish1 - start1;
+        System.out.println(timeConsumedMillis1);
+
+        System.out.println("------------");
+        rundomMath(homeTaskArr);
+        long start2 = System.currentTimeMillis();
+        homeTaskArr.sortInsert();
+        long finish2 = System.currentTimeMillis();
+        long timeConsumedMillis2 = finish2 - start2;
+        System.out.println(timeConsumedMillis2);
+
+    }
+
+    static void rundomMath(HomeTaskArr homeTaskArr){
```

Comment:
```suggestion
    static void randomMath(HomeTaskArr homeTaskArr){
```
также важно обнулять размер массива (`size = 0`) иначе для каждой сортировки мы только дополняем данными